qa/rpc-tests/getblockexpanded.py

```diff
@@ -0,0 +1,260 @@
+                    assert_equal(len(rpcDataByHash['matureCertificate']), 1)
+                    assert_equal(len(rpcDataByHeight['matureCertificate']), 1)
+                    assert_equal(rpcDataByHash['matureCertificate'][0], cert3)
+                    assert_equal(rpcDataByHeight['matureCertificate'][0], cert3)
+                    assert_equal(len(rpcDataByHashVerbosity['matureCertificate']), 1)
+                    assert_equal(len(rpcDataByHeightVerbosity['matureCertificate']), 1)
+                    assert_equal(rpcDataByHashVerbosity['matureCertificate'][0], cert_3_json)
+                    assert_equal(rpcDataByHeightVerbosity['matureCertificate'][0], cert_3_json)
+                else:
+                    assert_equal(len(rpcDataByHash['matureCertificate']), 0)
+                    assert_equal(len(rpcDataByHeight['matureCertificate']), 0)
+                    assert_equal(len(rpcDataByHashVerbosity['matureCertificate']), 0)
+                    assert_equal(len(rpcDataByHeightVerbosity['matureCertificate']), 0)
+            else:
+                assert_false('matureCertificate' in rpcDataByHash)
+                assert_false('matureCertificate' in rpcDataByHeight)
+                assert_false('matureCertificate' in rpcDataByHashVerbosity)
+                assert_false('matureCertificate' in rpcDataByHeightVerbosity)
+    
+        self.nodes[0].invalidateblock(new_epoch_block)
+        self.nodes[0].invalidateblock(cert4_block)
+        self.nodes[1].invalidateblock(new_epoch_block)
+        self.nodes[1].invalidateblock(cert4_block)
+        assert_equal(self.nodes[0].getblockcount(), 639)
+        assert_equal(self.nodes[1].getblockcount(), 639)
+
+        self.nodes[0].clearmempool()
+        self.nodes[1].clearmempool()
+        assert_equal(len(self.nodes[0].getrawmempool()),0)
+        assert_equal(len(self.nodes[1].getrawmempool()),0)
+        self.sync_all()
+
+        fake_block = self.nodes[0].generate(1)[0]
+        self.sync_all()
+
+        rpcDataByHeight = self.nodes[0].getblockexpanded("640")
+        rpcDataByHeightVerbosity = self.nodes[0].getblockexpanded("640", 2)
+        rpcDataByHash = self.nodes[0].getblockexpanded(rpcDataByHeight['hash'])
+        rpcDataByHashVerbosity = self.nodes[0].getblockexpanded(rpcDataByHeight['hash'],2)
+        assert_equal(len(rpcDataByHash['matureCertificate']), 0)
+        assert_equal(len(rpcDataByHeight['matureCertificate']), 0)
+        assert_equal(len(rpcDataByHashVerbosity['matureCertificate']), 0)
+        assert_equal(len(rpcDataByHeightVerbosity['matureCertificate']), 0)
+
+        self.nodes[0].invalidateblock(fake_block)
+        self.nodes[1].invalidateblock(fake_block)
+
+        self.nodes[0].sc_send_certificate(scid, epoch_number, quality,
+            epoch_cum_tree_hash, proof, amount_cert_4, FT_SC_FEE, MBTR_SC_FEE, CERT_FEE)
+        self.nodes[0].generate(1)[0]
+        self.nodes[0].generate(1)
```

Comment:
Can't we just generate two blocks?

```suggestion
        self.nodes[0].generate(2)
```